smart-contracts/src/contractSupport.ts

```diff
@@ -98,3 +98,20 @@ export function requiredEnvVar(name: string): string {
     throw `No setting for ${name} in environment`
   }
 }
+
+export interface DevEnvContracts {
+  cosmosBridge: CosmosBridge
+  bridgeBank: BridgeBank
+  bridgeRegistry: BridgeRegistry
+  rowanContract: BridgeToken
+}
+
+export async function buildDevEnvContracts(devEnv: DevEnvObject, hre: HardhatRuntimeEnvironment, factories: SifchainContractFactories): Promise<DevEnvContracts> {
```

Comment:
I think hre is not used here

Reply:
It's used in smart-contracts/test/watcher/watcher.ts